webapp/app/model/form_data.py

```diff
@@ -216,6 +223,7 @@ class FormDataDependencies(BaseModel):
     person_a_blind: Optional[bool]
     person_a_gehbeh: Optional[bool]
     person_a_has_disability: Optional[str]
+    person_a_requests_pauschbetrag: Optional[str]
```

Comment:
We should add a dependency that when `person_a_has_disability` is falsy (including `'no'`), `person_a_requests_pauschbetrag` is deleted.

Reply:
Done!